cloud-db/acl-overview/ios.md

```diff
@@ -0,0 +1,320 @@
+[publicDB performQuery:query completionHandler:^(NSArray *results, NSError *error) {
+    if (error) {
+        NSLog(@"error querying records: %@", error);
+        return;
+    }
+
+    // Successfully query records
+    SKYRecord *aNote = results[0];
+    aNote[@"content"] = @"Hello Wolrd!";
+    [publicDB saveRecord:aNote completion:^(SKYRecord * _Nullable record, NSError * _Nullable error) {
+        if (error) {
+            // You will get not Authenticated error here
+            NSLog(@"error saving records: %@", error);
+            return;
+        }
+
+        // You will NOT succeed
+    }];
+}];
+```
+
+```swift
+// not logged in as any user
+let query = SKYQuery(recordType: "note", predicate: nil)
+let publicDB = SKYContainer.default().publicCloudDatabase
+
+publicDB.perform(query) { (results, queryError) in
+    if queryError != nil {
+        print("Error: \(queryError?.localizedDescription)")
+        return
+    }
+
+    // Successfully query records
+    let aNote = notes[0] as SKYRecord
+    aNote["content"] = "Hello World!"
+    publicDB.save(aNote) { (results, saveError) in
+        if saveError != nil {
+            // You will get not Authenticated error here
+            print("Error: \(saveError?.localizedDescription)")
+            return
+        }
+
+        // You will NOT succeed
+    }
+}
+```
+
+If you want to customize these security settings, Skygear supports access control on records. You can use it to control whether a user can query, update or delete a record.
+
+Skygear uses [Access Control List][wiki-acl] (ACL) as the model of access control.
+An ACL is a list of Access Control Entry (ACE), each ACE describes which
+access rights a user target has to a record.
+
+Each ACE consists of:
+- **Access level**, which defines what actions can be performed (for example, read or write).
+- **User Target**, which defines who can perform the specified actions (for example, a specific user or group of users).
+
+
+These are the concepts you need to know:
+- Record ownership, **the owner has full access** to the record.
+- The 2 types of ACL in Skygear: [Record-based ACL][doc-record-acl] and [Field-based ACL][doc-field-acl].
+
+:::tips
+When you should use Record-based ACL or Field-based ACL?
+
+If every single record has the same ACL, that the same  have the same access rights to a record, you can use Record-based ACL.
+
+Otherwise, if a set of users have has different access rights for each field of a record, you would need to use Field-based ACL.
+:::
+
+Before diving into Record-based ACL and Field-based ACL, let's learn about the basic concepts of Skygear ACL.
+
+## ACL Access Level
+
+There are 3 access levels in Skygear ACL:
+
+- No Access
+- Read Write Access
+- Read Only Access
+
+This is how you can set it:
+
+```obj-c
+SKYAccessControl *acl = [SKYAccessControl emptyAccessControl];
+
+[acl setNoAccessForPublic];
+[acl setReadOnlyForPublic];
+[acl setReadWriteAccessForPublic];
+
+// and more...
```

Comment:
This `// and more....` is very strange to me... More wt..?

Reply:
Because I am just using Public as example, maybe just remove this line to prevent confusion.